spec/queries/project/subscriptions_pledged_value_query_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Project::SubscriptionsPledgedValueQuery do
+  describe ".call" do
+    let(:subscriptions_pledged_value) { Project::SubscriptionsPledgedValueQuery.new(project) }
+    let(:project) { create(:project) }
+
+    before { allow_any_instance_of(Project).to receive(:recurring?).and_return(true) }
```

Comment:
Do you really need that? I think you check if it's recurring before call the service, then here it's assumed it's a recurring project already, right?